MediaRecorder.bs

```diff
@@ -410,6 +401,42 @@ interface MediaRecorder : EventTarget {
   </table>
   </dd>
 
+  <dt><dfn method for="MediaRecorder"><code>replaceTrack(MediaStreamTrack track, MediaStreamTrack withTrack)</code></dfn></dt>
+  <dd>
+  When a {{MediaRecorder}} object&#8217;s {{replaceTrack()}} method is invoked,
+  the UA MUST run the following steps:
+  <ol>
+    <li>Let |recorder| be the {{MediaRecorder}} object on which the method was
+    invoked.</li>
+    <li>Let |track| be the method's first argument.</li>
+    <li>Let |withTrack| be the method's second argument.</li>
+    <li>If |recorder|'s {{state}} is {{inactive}}, throw an
+    {{InvalidStateError}} {{DOMException}} and abort these steps.</li>
+    <li>If |track| is not found in |recorder|'s [=[[Tracks]]=], throw a
```

Comment:
Does that mean that the individual `MediaStreamTrack` must be referenced directly? E.g., either 

`let track = mediaStreamDestination.stream.getAudioTracks() // track is passed to recorder, still the same track`

or 

`let track = recorder.stream.getAudioTracks()[0] // the same track passed to recorder referenced now using recorder.stream` 

?

Reply:
Yes, to know which track to replace in the general case of more than one track of a kind.

In the simple case (of one video one audio) a polyfill would be:
```js
recorder.replaceStream = async function (stream) {
  const old = recorder.stream;
  await recorder.replaceTrack(old.getAudioTracks()[0], stream.getAudioTracks()[0]);
  await recorder.replaceTrack(old.getVideoTracks()[0], stream.getVideoTracks()[0]);
  recorder.stream = stream;
}
```

Reply:
Is there a way to add this replaceTrack method for an individual project?